docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/redelegation-process.md

```diff
@@ -1,7 +1,44 @@
-# Redelegation Process
+---
+sidebar_position: 3
+title: Change Your Delegation to a New Operator
+---
 
-Redelegation is available for multiple scenarios for both LST and Native Restakers:
-- Restaker would like to undelegate from their current operator, restake, and then delegate to a new operator.
-- A user who accidentally queues a withdrawal, but would like to resume restaking and delegation without having to exit and re-enter their validators from the beacon chain.
+# Change Your Delegation to a New Operator
 
-Please see the following section [Undelegate from an Operator and Initiate Withdrawal](./undelegate-from-an-operator-and-initiate-withdrawal.md)
\ No newline at end of file
+The following use case is intended for a Restaker that would like to change their Delegated balance to a New Operator.
```

Comment:
totally nitpicking here, but Delegate seems to have some instances uppercase and some instances lowercase without much consistency

Reply:
@dabit3  I agree completely, maybe we should come up with some rules?

For now, my intention has been:
- Lower case for verbs like "restake", "delegate(d)", etc.
- Upper case for personas like "Restaker", "Operator".
- Upper case for actions like "click the Delegate button".

But if you have seen a better practice, let's adopt it.